data/remnant/remnant 2 cognizance.txt

```diff
@@ -394,41 +391,54 @@ mission "Remnant: Cognizance 9"
+			`Flight control gives you directions to a landing pad that already has a large pile of crates stacked next to it. As you land, workers swarm out to help load the equipment. Once everything is stowed, a young technician climbs aboard, stows their gear, and lets you know that everything is ready for departure. The Remnant can certainly make things happen quickly when projects start to move.`
 	on complete
 		payment 100000
 		conversation
 			`On the flight over, the technician joins you on the control deck to chat. She's a biotech specialist, adept at managing life-support systems in alien atmospheres, and a recent graduate from a Remnant educational program that seems to be similar to a university. Apparently she studied astrobiology and is part of a project aimed at supporting mutually exclusive environments in close proximity. She spends nearly an hour excitedly telling you the ins and outs of the station.`
 			`	As you approach <planet>, it occurs to you that the Remnant might appreciate you taking steps to maintain the secrecy of the station. Altering the <ship>'s path slightly, you swing down into <planet> in an oblique path then meander around the planet under the cloud layer until arriving at the station. Your passenger nods approvingly as she notices the flight path. "Secrecy is always good with projects like this. We appreciate the extra effort, and if anyone were to follow you, a path like that through the clouds would leave few signs to spot."`
-			`	The <ship> settles into the landing bay, and the technician quickly disembarks. Apparently there must have been other shipments while you were gone, as there are now a handful of people visible and quite a bit more equipment stacked everywhere. The station itself feels more alive, the air fresher. As before, you promptly receive notification of another payment of <payment> to your account.`
+			`	The <ship> settles into the landing bay, and the technician quickly disembarks. Apparently, there must have been other shipments while you were gone, as there are now a handful of people visible and quite a bit more equipment stacked everywhere. The station itself feels more alive, the air fresher. As before, you promptly receive notification of another payment of <payment> to your account.`
 
 
 
 mission "Remnant: Cognizance 11"
+	landing
 	name "Remnant Data Delivery"
 	description "Deliver a data crystal to the engineers on <planet>."
-	landing
 	source "Esquiline"
 	destination "Viminal"
 	to offer
-		has "Remnant: Cognizance 10: done"
+		or
+			has "Remnant: Cognizance 10: done"
+			has "Remnant: Cognizance 10B: done"
```

Comment:
I'm not entirely sure what is going on here, I'll have to re-read this later. (and this is as far as I can review at the moment).

That being said, should that "or" statement be checking for 10A or 10B instead of 10 or 10B? (since you replaced 10 with 10A above)?

Reply:
> That being said, should that "or" statement be checking for 10A or 10B instead of 10 or 10B? (since you replaced 10 with 10A above)?

I have split 10 into 10A and 10B.
10A is the trip from Esquiline to Caelian, 10B is the trip back to Esquiline, rather than Caelian being a stopover.
11 should require a person to have either done 10 (if they picked up the mission before this change) or 10B.